lib/tessel/mdns-list.js

```diff
@@ -0,0 +1,58 @@
+var mdns = require('mdns-js')
```

Comment:
totally, didn't know where to put it and was following old architecture. We should have a conversation about what the file structure is.